Week3/homework/exercise4.js

```diff
@@ -0,0 +1,20 @@
+"use strict"
+/*the grocery list should contain bananas and milk at first and then we should be adding other items*/
+let groceryItem = ["bananas", "milk"]
+let i = 0;
+/*the grocery list should have no more than 3 items, so each time I add another item, the first thing in the cart needs to be taken out*/
+function addToShoppingCart (item){
+    let moreitems = groceryItem.push(item) /*here we push the new item into the array*/
+    if (groceryItem.length > 3){
+        groceryItem.shift() /*and here we take the first item out since the length is already = 3*/
+
+    }
+    return `You bought ${groceryItem}!`
```

Comment:
- `const` instead of `let` and `moreItems` instead of `moreitems` variable please.
- I see you are not using semi-colons consistently. It's okay to either use them completely or not use them at all but try to follow a common pattern please 🙂

Reply:
Done.